orchard-ws/app/workflow/Router.scala

```diff
@@ -28,6 +28,7 @@ class Router @Inject() (ctrl: Controller) extends SimpleRouter {
         ) =>
       ctrl.filter(like, statuses, orderBy, order, page, perPage)
 
+    case GET(p"/$id/status") => ctrl.workflowStatus(id)
```

Comment:
instead of status, let's make the `GET` return the details. Basically everything in `WorkflowTable.R`